### ISSUE TITLE
Any suggestions on how to run this on ROS-Hydro ?

### ISSUE THREAD

**Issue by ktiwari9** [browny/hand-tracking#1]

**Comment by browny** [comment]
Maybe you could refer this: http://answers.opencv.org/question/19083/how-do-i-install-opencv-to-work-with-ros/

**Comment by ktiwari9** [comment]
Yeah i did go through this and I added those lines in CMakeList.txt but what I am not sure about is how do i compile and run the program to see some output ?
Please advise.

**Comment by ktiwari9** [comment]
I think the problem is that the version of opencv you have used is not compatible with ROS. Any suggestions on how i can fix this issue ?
